zkevm-circuits/src/evm_circuit/execution/balance.rs

```diff
@@ -114,8 +118,7 @@ impl<F: Field> ExecutionGadget<F> for BalanceGadget<F> {
         self.same_context.assign_exec_step(region, offset, step)?;
 
         let address = block.get_rws(step, 0).stack_value();
-        self.address_word
-            .assign(region, offset, Some(address.to_le_bytes()))?;
+        self.address.assign_u256(region, offset, address)?;
```

Comment:
Shouldn't this be `assign_h160(..., address.to_address())`?

Reply:
`assign_u256` is correct because here address already in u256 little endian 20 bytes